lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -21,7 +21,7 @@ public static void main(String[] args) {
    * @return True if the age corresponds to a voting age and false otherwise.
    */
   public static boolean canVote(int age) {
-    return false;
+    return age >= 18 ? true : false;
```

Comment:
Don't need the ternary part, will still work.

Reply:
removed ternary part!